stdlib/http/src/main/ballerina/http/Package.md

```diff
@@ -61,15 +61,16 @@ Here `upgradeService` is a `WebSocketService`.
 
 **onPing and onPong resources**: Received ping and pong messages are dispatched to these resources respectively as a `blob`
 
-**onIdleTimeout**: This resource is dispatched when idle timeout is reached. idleTimeout has to be configured by the user.
+**onIdleTimeout**: This resource is dispatched when idle timeout is reached. idleTimeout has to be configured by the user in WebSocket service configuration.
 
 **onClose**: This resource is dispatched when a close message is received.
 
-**onError**: This resource is dispatched when an error occurred in a particular WebSocket connection. This will always be followed by a connection closure.
+**onError**: This resource is dispatched when an error occurs in the WebSocket connection. This will always be followed by a connection closure with an appropriate WebSocket close frame and this resource is used to indicate the error to the user and to take post decisions if needed.
 
```

Comment:
Can we remove this part "and this resource is used to indicate the error to the user and to take post decisions if needed." because it seems to be confusing.